SS-common/MPU9250/SS_MPU9250.c

```diff
@@ -155,58 +181,105 @@ MPU_STATUS SS_MPU_math_scaled_accel(MPU9250 *mpu9250) {
     return MPU_OK;
 }
 
+int32_t data[6][2000];
+int16_t  data1[6];
 MPU_STATUS SS_MPU_calibrate(MPU9250 *mpu9250) {  //Device needs to be flat during calibration!!!
-    int16_t calibration_time = 2000;
-    uint16_t max_gyro_deviation = 15;
-    uint16_t max_accel_deviation = 200;
-    HAL_Delay(100);
+
+    int16_t calibration_time = 3;                // accel sometimes return 0 than proper data that why i read only 3 times
```

Comment:
To trzeba dojść dlaczego tak się dzieje, a nie znajdywać jakieś rozwiązanie na około

Reply:
Tylko chodzi o to że nie potrzebna jest tak duża liczba danych , po wstępnych testach funkcja zwraca wyniki w odchyleniu o ok 5